contracts/RewardEscrowV2.sol

```diff
@@ -432,8 +432,7 @@ contract RewardEscrowV2 is
         if (_earlyVestingFee > MAXIMUM_EARLY_VESTING_FEE) revert EarlyVestingFeeTooHigh();
         if (_earlyVestingFee < MINIMUM_EARLY_VESTING_FEE) revert EarlyVestingFeeTooLow();
         if (_deposit == 0) revert ZeroAmount();
-        uint256 minimumDuration = stakingRewards.cooldownPeriod();
-        if (_duration < minimumDuration || _duration > MAX_DURATION) revert InvalidDuration();
+        if (_duration == 0 || _duration > MAX_DURATION) revert InvalidDuration();
```

Comment:
**Question:** Are there any problems you could think of with entries of duration 1?

Reply:
No, there is no problem with entries of duration 1.
Did not write a specific scenario test as newly issued rewards are no longer be escrowed, but it is covered with some of the existing fuzz tests.